Makefile.js

```diff
@@ -708,6 +736,51 @@ target.checkLicenses = function() {
     });
 };
 
+target.checkGitCommit = function() {
+    var commitMsgs,
+        failed;
+
+    if (hasBranch("master")) {
```

Comment:
I still dont think this will work correctly. When Travis does a build, it checks out a single branch, so you might not have master to work with. I think you'd need to add an upstream origin, fetch from it, and compare to that too get accurate results.